lib/sinon.js

```diff
@@ -24,11 +24,8 @@ var apiMethods = {
     setFormatter: format.setFormatter,
 
     // fake timers
-    useFakeTimers: fakeTimers.useFakeTimers,
```

Comment:
So these lines that were hanging around from earlier versions essentially shadowed the new sandbox functions on the prototype. Good find!